lean/common.lean

```diff
@@ -431,16 +434,16 @@ def pp : Π{tp:type}, prim tp → string
 | ._ (bsf i) := "bsf " ++ i.pp
 | ._ (bsr i) := "bsr " ++ i.pp
 | ._ (bswap i) := "bswap " ++ i.pp
-| ._ zero := "0"
-| ._ one  := "1"
+| ._ zero := sexp.app "bit" ["0"]
+| ._ one  := sexp.app "bit" ["1"]
```

Comment:
I think we should just call these `false` and `true`.

Reply:
Do you mean changing `zero` and `one`, changing the output `(bit 0)`, or both?

Reply:
Sorry for the delay, I'd be inclined to do both, e..g. , `._ false := "false"`